torch/onnx/symbolic_opset9.py

```diff
@@ -2426,6 +2426,72 @@ def _convolution(
         return n
 
 
+@_onnx_symbolic("aten::_convolution_mode")
+@symbolic_helper.parse_args(
+    "v",
+    "v",
+    "v",
+    "is",
+    "s",
+    "is",
+    "i",
+)
+@_beartype.beartype
+def _convolution_mode(
```

Comment:
Why not extend existing `_convolution` function?

Reply:
Seems like aten::_convolution_mode is present in the IR graph as opposed to aten::_convolution in cases where padding is a string. Also the calling parameters for both these signatures are different.